src/Html.php

```diff
@@ -1145,21 +1146,25 @@ public static function includeHeader(
         $sector = 'none',
         $item = 'none',
         $option = '',
-        bool $add_id = true
+        bool $add_id = true,
+        bool $allow_iframe = false
```

Comment:
To make things clear from a security point of view.
```suggestion
        bool $allow_embedding = false
```